packages/charts/react-charting/src/components/ChartTable/ChartTable.base.tsx

```diff
@@ -55,22 +83,38 @@ export class ChartTableBase extends React.Component<IChartTableProps> {
               >
                 <thead>
                   <tr>
-                    {headers.map((header, idx) => (
-                      <th key={idx} className={classNames.headerCell} style={header?.style}>
-                        {header.value}
-                      </th>
-                    ))}
+                    {headers.map((header, idx) => {
+                      const style = { ...header?.style };
+                      const fg = style.color;
+                      const bg = style.backgroundColor;
+                      if (fg || bg) {
```

Comment:
>fg || bg [](http://example.com/codeflow?start=26&length=8)

what happens if none of fg or bg are provided in the schema. What values are set in that case.